db/migrate/20250816213210_add_audit_timestamps_to_collection_preferences.rb

```diff
@@ -0,0 +1,20 @@
+class AddAuditTimestampsToCollectionPreferences < ActiveRecord::Migration[7.1]
+  uses_departure! if Rails.env.staging? || Rails.env.production?
+
+  def change
+    change_table :collection_preferences, bulk: true do |t|
```

Comment:
Are you sure this interacts well with departure? We usually use `add_column` instead of bulk changing the table